src/herder/TransactionQueue.h

```diff
@@ -118,6 +119,7 @@ class TransactionQueue
 
     explicit TransactionQueue(Application& app, int pendingDepth, int banDepth,
                               int poolLedgerMultiplier);
+    ~TransactionQueue();
```

Comment:
Without this, Herder also needs the definition for `TxQueueLimiter` (I think it tries to inline the destructor?) and I'm trying to avoid bloating the include hierarchy with this stuff

Reply:
Yes, that's right. Hard to tell by looking at the code though (even as a PR review where I can see this was added alongside the `unique_ptr` I didn't notice, it'll be even harder when maintaining the code in the future), so please add a comment.